rdr_service/tools/tool_libs/backfill_coreminuspmtime.py

```diff
@@ -0,0 +1,77 @@
+from rdr_service import clock
+from rdr_service.dao.participant_summary_dao import ParticipantSummaryDao
+from rdr_service.logic.enrollment_info import EnrollmentCalculation, EnrollmentDependencies
+from rdr_service.model.participant_summary import ParticipantSummary
+from rdr_service.participant_enums import EnrollmentStatus
+from rdr_service.repository.questionnaire_response_repository import QuestionnaireResponseRepository
+from rdr_service.services.system_utils import list_chunks, min_or_none
+from rdr_service.tools.tool_libs.tool_base import cli_run, ToolBase
+
+tool_cmd = 'backfill-coreminuspmtime'
+tool_desc = 'Backfill CoreMinusPMTime values that had been incorrectly updated'
+
+
+class BackfillCoreMinusPMTime(ToolBase):
+    def run(self):
+        super().run()
+        with self.get_session() as session:
+            summary_list = session.query(ParticipantSummary).all()
```

Comment:
suggestion: I'm finding with the the enrollment status backfill that it takes a while and I sometimes lose connection with the database (whether it's my computer going to sleep or some other cause). Something I've done is ordered the participants by their id and logged what the last id was of each subset. That way if it fails because of a connection issue you can add a filter for the participant id and start from the same spot rather than from the beginning.